osu.Game/Overlays/MusicController.cs

```diff
@@ -385,15 +387,18 @@ public bool AllowTrackAdjustments
         /// </remarks>
         public void ResetTrackAdjustments()
         {
+            // todo: we probably want a helper method rather than this.
             CurrentTrack.RemoveAllAdjustments(AdjustableProperty.Balance);
             CurrentTrack.RemoveAllAdjustments(AdjustableProperty.Frequency);
             CurrentTrack.RemoveAllAdjustments(AdjustableProperty.Tempo);
             CurrentTrack.RemoveAllAdjustments(AdjustableProperty.Volume);
```

Comment:
This could now probably be replaced with:
```
if (modTrackAdjustments != null)
{
    CurrentTrack.UnbindAdjustments(modTrackAdjustments);
    modTrackAdjustments = null;
}
```

Reply:
Well that's what I had hoped for, but `CurrentTrack` could have adjustments applied directly from other screens like `Player`. Your other suggestion in https://github.com/ppy/osu/pull/18211#discussion_r870983474 may actually work best (defining an `AudioAdjustments` property exposed in `Mod` and lets usages bind to it), but maybe for another day?